src/_constants/ActionTypes.js

```diff
@@ -40,3 +40,6 @@ export const SIGNIN_FAILED = 'SIGNIN_FAILED';
 
 export const SERVER_DATA_ACCOUNT_LIMITS = 'SERVER_DATA_ACCOUNT_LIMITS';
 export const SERVER_DATA_ACCOUNT_SETTINGS = 'SERVER_DATA_ACCOUNT_SETTINGS';
+
+export const NEXT_VIDEO = 'NEXT_VIDEO';
+export const PREVIOUS_VIDEO = 'PREVIOUS_VIDEO';
```

Comment:
hmm, so now we do not want to have the video in the apps? and user can swipe left or right to view next?